### DISCUSSION TITLE
adding cache-control header does not work

### DISCUSSION THREAD

**Discussion by dario-digregorio** [fleaflet/flutter_map#1472] [Q&A] [answered]
Hi, 
I added the Cache-Control header to a Tile Provider:
```dart
tileProvider: NetworkTileProvider(
  headers: {
    'Cache-Control': 'public, max-age=31536000',
  },
),
```
Looking at the flutter devtool network tab it seems like they are being sent too. I also get a x-cache-key as a response. But Network Tools still shows me that it is doing request although it is calling the same tiles.

Am I missing here something?

**Comment by JaffaKetchup** [comment, marked as answer]
Hi @dario-digregorio,
Unless Dart is doing something in the background, we don't make any effort to cache. For information about caching, please see https://docs.fleaflet.dev/usage/layers/tile-layer/tile-providers#caching-and-bulk-downloading.

**Reply by dario-digregorio** [reply]
I found the issue too:

https://github.com/flutter/flutter/issues/11595
